modules/autoscalinggroup/main.tf

```
@@ -87,59 +75,88 @@ locals {
     "GroupTotalInstances",
   ]
 
-  asg_tags = ["${concat(
-      list(map("key", "Name", "value", local.name, "propagate_at_launch", true)),
-      local.tags_asg_format
-   )}"]
+  asg_tags = [
+    {
+      "key"                 = "Name"
+      "value"               = local.name
+      "propagate_at_launch" = true
+    },
+  ]
 }
 
 # An ASG where every node is identical
 resource "aws_autoscaling_group" "homogenous" {
-  count = "${var.create && !var.enable_mixed_cluster ? 1 : 0 }"
-  name  = "${local.name}"
+  count = var.create && false == var.enable_mixed_cluster ? 1 : 0
```

Either retain the `not` operator

```suggestion
  count = var.create && ! var.enable_mixed_cluster ? 1 : 0
```
or use the normal ordering for ease of reading:
```suggestion
  count = var.create && var.enable_mixed_cluster == false ? 1 : 0
```